dev/release/verify-release-candidate.sh

```diff
@@ -225,6 +225,29 @@ test_js() {
   popd
 }
 
+test_rust() {
+  # install rust toolchain in a similar fashion like test-miniconda
+  export RUSTUP_HOME=`pwd`/test-rustup
+  export CARGO_HOME=`pwd`/test-rustup
+
+  curl https://sh.rustup.rs -sSf | sh -s -- -y
+  source $RUSTUP_HOME/env
+
+  # build and test rust
+  pushd rust
+
+  # raises on any formatting errors (disabled, because RC1 has a couple)
```

Comment:
Here is the [fix](https://github.com/apache/arrow/commit/889e1e6cde9455d1ed1dd6b827909fbeec0b1726)